File: create_flags.py

import psycopg2
import os
from datetime import datetime

def create_flags():
    try:
        conn = psycopg2.connect(
            host='host.docker.internal',
            port=5432,
            user='pioneer',
            password='pioneer123',
            database='pioneer'
        )
        
        cursor = conn.cursor()
        
        flags = [
            ('dashboard_enabled', 'Enable main dashboard access', True),
            ('login_enabled', 'Enable login functionality', True),
            ('registration_enabled', 'Enable user registration', True),
            ('otp_verification_enabled', 'Enable OTP verification', True)
        ]
        
        for title, description, is_active in flags:
            cursor.execute('''
                INSERT INTO flags (title, description, is_active, version, rollout, updated_at, created_at) 
                VALUES (%s, %s, %s, 1, 100, %s, %s)
                ON CONFLICT (title) DO NOTHING
            ''', (title, description, is_active, datetime.now(), datetime.now()))
            print(f'Created flag: {title}')
        
        conn.commit()
        print('All flags created successfully!')
        
    except Exception as e:
        print(f'Error: {e}')
    finally:
        if 'cursor' in locals():
            cursor.close()
        if 'conn' in locals():
            conn.close()

if __name__ == '__main__':
    create_flags()

File: create_flags.js

const { Client } = require('pg');

const client = new Client({
  host: 'host.docker.internal',
  port: 5432,
  user: 'pioneer',
  password: 'pioneer123',
  database: 'pioneer'
});

async function createFlags() {
  try {
    await client.connect();
    
    const flags = [
      { title: 'dashboard_enabled', description: 'Enable main dashboard access', is_active: true },
      { title: 'login_enabled', description: 'Enable login functionality', is_active: true },
      { title: 'registration_enabled', description: 'Enable user registration', is_active: true },
      { title: 'otp_verification_enabled', description: 'Enable OTP verification', is_active: true }
    ];
    
    for (const flag of flags) {
      await client.query(
        'INSERT INTO flags (title, description, is_active, version, rollout, updated_at, created_at) VALUES (cd /Users/manish/Documents/GitHub/zero/agentmitra && echo "
const { Client } = require('pg');

const client = new Client({
  host: 'host.docker.internal',
  port: 5432,
  user: 'pioneer',
  password: 'pioneer123',
  database: 'pioneer'
});

async function createFlags() {
  try {
    await client.connect();
    
    const flags = [
      { title: 'dashboard_enabled', description: 'Enable main dashboard access', is_active: true },
      { title: 'login_enabled', description: 'Enable login functionality', is_active: true },
      { title: 'registration_enabled', description: 'Enable user registration', is_active: true },
      { title: 'otp_verification_enabled', description: 'Enable OTP verification', is_active: true }
    ];
    
    for (const flag of flags) {
      await client.query(
        'INSERT INTO flags (title, description, is_active, version, rollout, updated_at, created_at) VALUES ($1, $2, $3, 1, 100, NOW(), NOW()) ON CONFLICT (title) DO NOTHING',
        [flag.title, flag.description, flag.is_active]
      );
      console.log(\`Created flag: \${flag.title}\`);
    }
    
    console.log('All flags created successfully');
  } catch (error) {
    console.error('Error creating flags:', error);
  } finally {
    await client.end();
  }
}

createFlags();
" > create_flags.js, , , 1, 100, NOW(), NOW()) ON CONFLICT (title) DO NOTHING',
        [flag.title, flag.description, flag.is_active]
      );
      console.log(`Created flag: ${flag.title}`);
    }
    
    console.log('All flags created successfully');
  } catch (error) {
    console.error('Error creating flags:', error);
  } finally {
    await client.end();
  }
}

createFlags();

